ChangeLog.md

```diff
@@ -135,7 +135,10 @@
   (David Coeurjolly, [#1446](https://github.com/DGtal-team/DGtal/pull/1446))
   - Removes the check for CPP11 when building WITH_ITK
   (Pablo Hernandez-Cerdan, [#1453](https://github.com/DGtal-team/DGtal/pull/1453))
-
+  - Fix apple clang  compilation issue with a workaround to the
+    ConstIteratorAdapter that do not satisfy the _is_forward concept of the STL:
```

Comment:
```suggestion
    ConstIteratorAdapter class that does not satisfy the _is_forward concept of the STL:
```

Reply:
thanks, I miss the commit suggestion since my emacs was open on it but that looks nice!